ibm/mas_devops/roles/ocp_provision/README.md

```diff
@@ -31,6 +31,12 @@ Flag that determines if GPU worker nodes should be added during cluster creation
 - Environment Variable: `OCP_PROVISION_GPU`
 - Default Value: `false`
 
+### gpu_wpool_name
```

Comment:
A bit of a nit-pick but could we call the var `gpu_workerpool_name` rather than `gpu_wpool_name`? I don't think we have shorted names like that in other places. Environment variable name would change aswell

Reply:
okay, I will do that. :)